tests/queries/0_stateless/00080_show_tables_and_system_tables.sql

```diff
@@ -1,25 +1,24 @@
--- Tags: no-parallel
 
-DROP DATABASE IF EXISTS test_show_tables;
+DROP DATABASE IF EXISTS {CLICKHOUSE_DATABASE:Identifier};
 
-CREATE DATABASE test_show_tables;
+CREATE DATABASE {CLICKHOUSE_DATABASE:Identifier};
 
-CREATE TABLE test_show_tables.A (A UInt8) ENGINE = TinyLog;
-CREATE TABLE test_show_tables.B (A UInt8) ENGINE = TinyLog;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.A (A UInt8) ENGINE = TinyLog;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.B (A UInt8) ENGINE = TinyLog;
 
-SHOW TABLES from test_show_tables;
+SHOW TABLES from {CLICKHOUSE_DATABASE:Identifier};
 SHOW TABLES in system where engine like '%System%' and name in ('numbers', 'one');
 
-SELECT name, toUInt32(metadata_modification_time) > 0, engine_full, create_table_query FROM system.tables WHERE database = 'test_show_tables' ORDER BY name FORMAT TSVRaw;
+SELECT name, toUInt32(metadata_modification_time) > 0, engine_full, create_table_query FROM system.tables WHERE database = currentDatabase() ORDER BY name FORMAT TSVRaw;
 
 CREATE TEMPORARY TABLE test_temporary_table (id UInt64);
 SELECT name FROM system.tables WHERE is_temporary = 1 AND name = 'test_temporary_table';
 
-CREATE TABLE test_show_tables.test_log(id UInt64) ENGINE = Log;
-CREATE MATERIALIZED VIEW test_show_tables.test_materialized ENGINE = Log AS SELECT * FROM test_show_tables.test_log;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.test_log(id UInt64) ENGINE = Log;
+CREATE MATERIALIZED VIEW {CLICKHOUSE_DATABASE:Identifier}.test_materialized ENGINE = Log AS SELECT * FROM {CLICKHOUSE_DATABASE:Identifier}.test_log;
 SELECT dependencies_database, dependencies_table FROM system.tables WHERE name = 'test_log';
 
-DROP DATABASE test_show_tables;
+DROP DATABASE {CLICKHOUSE_DATABASE:Identifier};
```

Comment:
Do we have to drop the database? I think it is fine to leave it as is, as the name is a random generated name, it shouldn't interfere with anything else. That is true for other tests too where you introduced `{CLICKHOUSE_DATABASE:Identifier}`.

Reply:
We don't have to, but it's okay to drop it. I just replaced all occurrences of `test_show_tables`